packages/directory/addon/templates/components/dir-table-filter.hbs

```diff
@@ -1,7 +1,11 @@
 {{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 {{#basic-dropdown classNames="dir-table-filter" as |dd|}}
   {{#dd.trigger class="dir-table-filter__trigger"}}
-    {{capitalize selectedFileType}}
+    <span 
+      class="dir-table-filter__trigger-type" 
+    >
```

Comment:
Can we put this as just `<span class="dir-table-filter__trigger-type">`

Reply:
Yes, I forgot to format it after removing onclick and role.